docs/components/PageHeader.js

```diff
@@ -52,49 +54,50 @@ export default function PageHeader({
         },
       }}
     >
-      <Box marginBottom={2}>
-        <Flex alignItems="baseline" direction="row" gap={2} justifyContent="between" wrap>
-          <Heading>
-            {name}{' '}
-            {badge === 'pilot' ? (
-              <Tooltip
-                inline
-                text={`This is the initial version of ${name}, and additional (non-breaking) functionality is planned for the future. Any feedback is greatly appreciated!`}
+      <Flex direction="column" gap={defaultCode ? 8 : 0}>
+        <Flex direction="column" gap={2}>
+          <Flex alignItems="baseline" direction="row" gap={2} justifyContent="between" wrap>
+            <Heading>
+              {name}{' '}
+              {badge === 'pilot' ? (
+                <Tooltip
+                  inline
+                  text={`This is the initial version of ${name}, and additional (non-breaking) functionality is planned for the future. Any feedback is greatly appreciated!`}
+                >
+                  <Badge text="Pilot" position="top" />
+                </Tooltip>
+              ) : null}
+              {badge === 'deprecated' ? (
+                <Tooltip inline text="This component is deprecated and will be removed soon.">
+                  <Badge text="Deprecated" position="top" />
+                </Tooltip>
+              ) : null}
+            </Heading>
+
+            {showSourceLink && (
+              <a
+                href={sourceLink}
+                onClick={() => trackButtonClick('View source on GitHub', sourcePathName)}
+                target="blank"
               >
-                <Badge text="Pilot" position="top" />
-              </Tooltip>
-            ) : null}
-            {badge === 'deprecated' ? (
-              <Tooltip inline text="This component is deprecated and will be removed soon.">
-                <Badge text="Deprecated" position="top" />
-              </Tooltip>
-            ) : null}
-          </Heading>
+                <Text underline>View source on GitHub</Text>
+              </a>
+            )}
+          </Flex>
 
-          {showSourceLink && (
-            <a
-              href={sourceLink}
-              onClick={() => trackButtonClick('View source on GitHub', sourcePathName)}
-              target="blank"
-            >
-              <Text underline>View source on GitHub</Text>
-            </a>
-          )}
+          {description && <Markdown text={description} />}
+          {slimBanner ? slimBanner : null}
```

Comment:
nit: add `null` as a default value for slimBanner and this can just be
```js
{slimBanner}
```

Reply:
DONE